src/main/java/org/folio/rest/impl/PutOrderLineByIdHelper.java

```diff
@@ -51,7 +50,10 @@ public PutOrderLineByIdHelper(Map<String, String> okapiHeaders, Handler<AsyncRes
    */
   public void updateOrderLine(String orderId, PoLine compOrderLine) {
     getPoLineByIdAndValidate(orderId, compOrderLine.getId())
-      .thenCompose(lineFromStorage -> updateOrderLine(compOrderLine, lineFromStorage))
+      .thenCompose(lineFromStorage -> {
+        compOrderLine.setPoLineNumber(lineFromStorage.getString(PO_LINE_NUMBER));
```

Comment:
Please add a comment to clarify that this is to override PO line number in the request with one from the storage, because it's not allowed to change it during PO line update.

Reply:
done